Gopkg.toml

```diff
@@ -93,3 +93,8 @@
 
 [[constraint]]
   name = "gopkg.in/yaml.v2"
+
+
+[[constraint]]
+  name = "github.com/kolide/launcher"
+  branch = "master"
```

Comment:
Enforcing master for now because both projects are moving in sync. There's a fix in master for encoding log types which is not in the current launcher release.